src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1630,16 +1630,6 @@ public override BoundNode VisitUsingStatement(BoundUsingStatement node)
 
                     default:
                         Debug.Assert(local.DeclarationKind == LocalDeclarationKind.UsingVariable);
-                        int slot = GetOrCreateSlot(local);
-                        if (slot >= 0)
-                        {
-                            SetSlotAssigned(slot);
-                            NoteWrite(local, value: null, read: true);
-                        }
-                        else
-                        {
-                            Debug.Assert(_emptyStructTypeCache.IsEmptyStructType(local.Type));
-                        }
                         break;
```

Comment:
Reasonable, but I would prefer not to do it in only one place where this pattern occurs. So I'm not going to do it in this PR.

---
In reply to: [78840194](https://github.com/dotnet/roslyn/pull/13807#discussion_r78840194) [](ancestors = 78840194)